manage/ships/repair.php

```diff
@@ -8,9 +8,7 @@
 
 $container = new Container($configuration);
 $shipLoader = $container->getShipLoader();
-// get our ship
-$ship = $shipLoader->findOneById($id);
-$ship->repairShip(); // updated currenthealth on the ship obj
-$shipLoader->repairShip($ship);
 
-header('Location: /manage/ships/index.php');
+$shipLoader->repairShip($id);
```

Comment:
Should still pass in the object.

Also might be good to show an error message if the ship is not found.

Reply:
Added logic to return to the view page without an ID and display "ship not found" and allows user to go back to previous pages.